DOCS/kratos-how-to-use-local-idp.md

```diff
@@ -0,0 +1,49 @@
+The Identity Platform can be run both in identity broker and identity provider mode.
+This guide explains how to use the local identity provider or disable it.
+
+The built-in identity provider is enabled by default in Identity Platform version 0.3.
+If you wish to disable that feature and allow your users to authenticate with external providers only, run:
+```
+juju config kratos enable_local_idp=False
+```
+
+Otherwise, the sign in screen will offer to log in with the internal identity provider as well as any configured external providers:
+
+![Alt]( https://raw.githubusercontent.com/canonical/canonical-identity-platform-docs/main/Diagram_sources/identity_platform_sign_in_page.png "IAM Login UI")
+
+## Enforce multi-factor authentication
+By default, all users created as part of the internal identity provider are required to set up time-based one-time password (TOTP) multi-factor authentication (MFA)
+by connecting with an authenticator app of their choice (e.g. Google Authenticator) on first logon and continue to use it on subsequent logons.
+
+![Alt]( https://raw.githubusercontent.com/canonical/canonical-identity-platform-docs/main/Diagram_sources/idp_secure_account_mfa.png "Set up MFA")
+
+It is also recommended that each user generates backup codes, so that they can be used as a fallback 2fa method in case the TOTP device is unavailable:
```

Comment:
How can I generate backup codes? When I create a user via the juju cli, I am asked to register an authenticator, but I can't find the option for generating the backup codes

Reply:
Shouldn't this somehow be reflected in the docs? Or simply remove this until there is a user friendly way to access the page?

Reply:
Added info how to access those settings